@navikt/core/react/src/show-more/ShowMore.tsx

```diff
@@ -0,0 +1,117 @@
+import React, { forwardRef, useState } from "react";
+import cl from "clsx";
+import { ChevronDownIcon, ChevronUpIcon } from "@navikt/aksel-icons";
+import { Button, OverridableComponent } from "..";
+
+interface ShowMoreBaseProps
+  extends Omit<React.HTMLAttributes<HTMLDivElement>, "onClick"> {
+  children: React.ReactNode;
+  /**
+   * Opens component if 'true', collapses if 'false'.
+   * Using this prop removes automatic control of open-state.
+   */
+  open?: boolean;
```

Comment:
Trenger man en controlled-state for denne i v1? Har det i expansioncard og accordion etc, men vært interresant å se om noe faktisk trenger featuren

Reply:
Enig, fjernet.